.github/workflows/ci.yaml

```
@@ -1,31 +1,37 @@
-name: 'Terraform fmt and validate'
-
-# Terraform linting & formatting checks;
-# Required files & documentation checks;
-# Required metadata checks (e.g. terraform.required_version);
-# Security checks, including aspects like secrets scanning & approved providers used.
+name: Terraform CI
 
 on:
-  - pull_request
-
-env:
-  tf_version: 'latest'
-  tf_working_dir: '.'
+- push
+- pull_request
 
 jobs:
-  terraform:
-    name: 'Terraform'
+  matrixify:
+    name: Matrixify
+    outputs:
+      matrix: ${{ steps.search.outputs.matrix }}
     runs-on: ubuntu-latest
 
     steps:
-      - name: 'Checkout'
-        uses: actions/checkout@master
+    - name: Get Changed Terraform directories
+      id: search
+      uses: mozilla/tf-actions/matrixify@OPST-126
```

```suggestion
      uses: mozilla/tf-actions/matrixify@main
```
to change once blocking PR in; left as example of CI for review of blocking Pr.